README.md

```diff
@@ -71,605 +137,84 @@ SYNOPSIS
-![Throughput plots light theme](/figures/loop_time-512-keydb_light.png#gh-light-mode-only "Throughput scaling for 512 node KeyDB DB")
-
-### Result analysis
-
-> :warning: from the above plots, it is clear that there is a performance decrease at 64 and 128 KiB, which is visible in all cases,
-but is most relevant for large DB node counts and for KeyDB. We are currently investigating this behavior, as we are not exactly
-sure of what the root cause could be.
-
-A few interesting points:
-
- 1. Client connection time: KeyDB connects client MUCH faster than base Redis. At this time, we
-    are not exactly sure why, but it does. So much so, that if you are looking to use the SmartRedis
-    clients in such a way that you will be disconnecting and reconnecting to the database, you
-    should use KeyDB instead of Redis with SmartSim.
-
- 2. In general, according to the throughput scaling tests, KeyDB has roughly up to 2x the throughput
-    of Redis and reaches a maximum throughput of ~125 Gb/s, whereas we could not get Redis to achieve
-    more than ~90 Gb/s.
-
- 3. KeyDB seems to handle higher numbers of clients better than Redis does.
-
- 4. There is an evident bottleneck on throughput around 128 kiB
-
-
-## Advanced Performance Tips
-
-There are a few places users can look to get every last bit of performance.
-
- 1. TCP settings
- 2. Database settings
-
-The communication goes over the TCP/IP stack. Because of this, there are
-a few settings that can be tuned
-
- - ``somaxconn`` - The max number of socket connections. Set this to at least 4096 if you can
- - ``tcp_max_syn_backlog`` - Raising this value can help with really large tests.
-
-The database (Redis or KeyDB) has a number of different settings that can increase
-performance.
-
-For both Redis and KeyDB:
-  - ``maxclients`` - This should be raised to well above what you think the max number of clients will be for each DB shard
-  - ``threads-per-queue`` - can be set in ``Orchestrator()`` init. Helps with GPU inference performance (set to 4 or greater)
-  - ``inter-op-threads`` - can be set in ``Orchestrator()`` init. helps with CPU inference performance
-  - ``intra-op-threads`` - can be set in ``Orchestrator()`` init. helps with CPU inference performance
-
-For Redis:
-  - ``io-threads`` - we set to 4 by default in SmartSim
-  - ``io-use-threaded-reads`` - We set to yes (doesn't usually help much)
-
-For KeyDB:
-  - ``server-threads`` - Makes a big difference. We use 8 on HPC hardware. Set to 4 by default.
-
+results/
+├─ inference-standard-scaling/
+│  ├─ run-2023-07-17-13:21:17/
+│  │  ├─ database/
+│  │  ├─ infer-sess-cpp-N4-T18-DBN4-DBCPU8-ITER100-DBTPQ8-80e4/
+│  │  ├─ infer-sess-fortran-N4-T18-DBN4-DBCPU8-ITER100-DBTPQ8-f8a6/
+│  │  ├─ run.cfg
+│  │  ├─ scaling-2023-07-19.log
+│  ├─ run-2023-07-19-11:33:57
+│  ├─ run-2023-07-19-11:40:08
+```
+
+Within each run folder there is a subset of files that will be useful to you.
+There is a file named `run.cfg` that contains the flag
```

Comment:
Turn these into a bulleted list

Reply:
I added notes to the bash script instead